### ISSUE TITLE
Style/UnneededCondition should exclude elsif

### ISSUE THREAD

**Issue by jodosha** [rubocop-hq/rubocop#5954]
`Style/UnneededCondition` cop should exclude `elsif`.

--------

## Expected behavior

```ruby
url = if params[:foo]
  params[:foo]
elsif some_other_condition?
  do_this
  interpolate_url
end
```

`Style/UnneededCondition` should ignore this condition because it's hard to write a one liner of the code above.

## Actual behavior

Rubocop reports a violation for the first line of the snippet.

## Steps to reproduce the problem

Please use the snippet above.

## RuboCop version

```
$ rubocop -V
0.57.0 (using Parser 2.5.1.0, running on ruby 2.4.3 x86_64-darwin16)
```

**Comment by bquorning** [comment]
Autocorrect also doesn’t work in this case:

```rb
url = if params[:foo]
  params[:foo]
elsif some_other_condition?
  do_this
  interpolate_url
end
```

Fixed using `rubocop --only Style/UnneededCondition -a test.rb` becomes…

```rb
url = params[:foo] || elsif some_other_condition?
  do_this
  interpolate_url
```

which is no longer valid Ruby.

**Comment by koic** [comment]
This case was changed to be ignored by PR #5957. It will not be autocorrected by that.

**Comment by jodosha** [comment]
Thank you all for the quick fix. 🎩
